js/focus_listener.js

```diff
@@ -1,7 +1,7 @@
 (function () {
   'use strict';
 
-  var windowFocused;
+  var windowFocused = false;
```

Comment:
Random changes! :0)

Reply:
Yes, but less randomization than opening a new PR 😉

Reply:
On a more serious note, I consider this part of continuous improvement based on the [broken window theory](https://blog.codinghorror.com/the-broken-window-theory/). But let me know if you prefer separate PRs for any orthogonal change. I was thinking we could allow smaller improvements within reason to reduce the overhead of separate branch creation, PR opening, CR, testing, etc.